mllib/src/main/scala/org/apache/spark/ml/feature/FeatureHasher.scala

```diff
@@ -91,8 +91,7 @@ class FeatureHasher(@Since("2.3.0") override val uid: String) extends Transforme
   /**
    * Numeric columns to treat as categorical features. By default only string and boolean
    * columns are treated as categorical, so this param can be used to explicitly specify the
-   * numerical columns to treat as categorical. Note, the relevant columns must also be set in
-   * `inputCols`.
+   * numerical columns to treat as categorical.
```

Comment:
This is still 'required' right? we're not making it an error, but it won't have any effect if not in inputCols.